client/src/config.js

```diff
@@ -1,12 +1,12 @@
 export const AmplifyConfig = {
-    apiGateway: {
-      REGION: "us-west-1",
-      URL: "https://v7mi6n82x1.execute-api.us-west-1.amazonaws.com/dev"
-    },
-    cognito: {
-      REGION: "us-west-1",
-      USER_POOL_ID: "us-west-1_YqFXItxNG",
-      APP_CLIENT_ID: "2jfr5crb2smk47k480unvd1rem",
-      IDENTITY_POOL_ID: "us-west-1:ace65909-a8da-405b-83b6-5362c269fba5"
-    }
-  };
\ No newline at end of file
+  apiGateway: {
+    REGION: "us-west-1",
+    URL: "https://ynm2dnizf6.execute-api.us-west-1.amazonaws.com/develop"
```

Comment:
why do we need "develop" endpoints when we already have "dev"?

Reply:
Was having problems. There's an issue to move these to a .env file